tests/lib/rules/require-valid-default-prop.js

```diff
@@ -259,8 +257,7 @@ ruleTester.run('require-valid-default-prop', rule, {
         parser: require('@typescript-eslint/parser'),
         ecmaVersion: 6,
         sourceType: 'module'
-      },
-      errors: errorMessage('function')
+      }
```

Comment:
These changes are not really related to your changes, right? It was just missed before and you cleaned it up here?

Reply:
That's right 😄